src/http/start.ts

```diff
@@ -0,0 +1,32 @@
+import Bootstrap from '../charging-station/Bootstrap';
+import express from 'express';
+
+const app = express();
+
+app.get('/', (request, response) => {
+  response.send(`
+ 
+ 
+  <form>
+  <input type="button" onclick="window.location.href='/start';" value="Start" />
+  <input type="button" onclick="window.location.href='/stop';" value="Stop" />
+</form>
+
+  `);
+});
+
+app.get('/start', (request, response, next) => {
+  Bootstrap.getInstance().start().catch(next);
+  console.log('*** started');
+  response.send('<b>Started</b><br/><a href="/">Return to Top</a>');
+});
+
+app.get('/stop', (request, response, next) => {
+  Bootstrap.getInstance().stop().catch(next);
+  console.log('*** stopped');
+  response.send('<b>Stopped</b><br/><a href="/">Return to Top</a>');
+});
+
+app.listen(process.env.PORT ?? 8080, () =>
+  console.log(`Listening on port: ${process.env.PORT ?? 8080}`)
+);
```

Comment:
I think the correct path for a UI is under an independent path like ui/httpd with its own package.json, etc. 
For now, could you please rename the path to src/ui/httpd?

Reply:
You just push the changes under the branch under your cloned repo referenced in that PR.

Reply:
You  still need to: 

+ fix GitHub CI actions issues.

Reply:
I upgraded node.js to v16.14.2 and did the npm install like you suggested.  I committed the package-lock.json, but not sure if that should be done.

Thanks for your patience.